### PR TITLE
[FEAT] CCM 15393: Adding information about CAG for GCP

### PR DESCRIPTION
**Pull request by joeyouss**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [x] Issue
- [ ] Feature
- [] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ At a high level, Cloud Custodian integrates seamlessly with services, features,
 
 
 :::info important
-Currently, Harness supports AWS and Azure cloud governance.
+Currently, Harness supports GCP, AWS and Azure cloud governance.
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md:19]: Keep it sorted: AWS, Azure, GCP.

File: docs/cloud-cost-management/get-started/onboarding-guide/set-up-cost-visibility-for-azure.md [hunk 2]
```diff
@@ -166,6 +166,7 @@ Billing export is used to get insights into your cloud infrastructure and Azure
...
 | **Azure Inventory Management** (Optional)| This feature provides visibility into your Azure VM inventory dashboard and metrics dashboard. The insights provided by inventory management can be used by finance teams to understand resource utilization across the board.|
 | **Azure optimization using AutoStopping rules** (Required for AutoStopping Rules)| This feature allows you to enable Intelligent Cloud AutoStopping for your Azure instances with a simple one-time setup. For more information, go to **Create AutoStopping Rules for Azure**.<ul><li>Orchestrate GCE VMs based on idleness</li><li>Set dependencies between VMs</li><li>Granular savings visibility</li><li>Simple one-time setup</li></ul>|
+| **Cloud Governance** (Optional)              | This feature allows you to optimize your cloud spend and avoid unnecessary costs by rightsizing resources and decommissioning unused instances. For more information, see [Asset governance](../../5-use-ccm-cost-governance/asset-governance/1-asset-governance.md). <ul><li>Asset Management (EC2, EBS, RDS, S3)</li><li>Automated Actions</li></ul>
```
Comment (iamanmolm) [inline docs/cloud-cost-management/get-started/onboarding-guide/set-up-cost-visibility-for-azure.md:169]: Add example like ec2, ebs, azure.vm, gcp.instance.

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/static/filter-evalaution-rules.png
Comment (rajarshee-harness) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/static/filter-evalaution-rules.png:1]: @iamanmolm This and another screenshot has the GCP Project Name. Should it be removed or it's fine?

Comment (iamanmolm) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/static/filter-evalaution-rules.png:1]: Yeah we should hide any internal account information to be shown on docs.

Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/static/filter-evalaution-rules.png:1]: Thank you for the heads up!

File: docs/cloud-cost-management/get-started/onboarding-guide/set-up-cost-visibility-for-aws.md [hunk 3]
```diff
@@ -144,7 +144,7 @@ Details about the features are listed below. Note that the permissions required
...
+| **Cloud Governance** (Optional)              | This feature allows you to optimize your cloud spend and avoid unnecessary costs by rightsizing resources and decommissioning unused instances. For more information, see [Asset governance](../../5-use-ccm-cost-governance/asset-governance/1-asset-governance.md). <ul><li>Asset Management (EC2, EBS, RDS, Azure VM, GCP instance, S3)</li><li>Automated Actions</li></ul>                                                                                                                                                                                                                                                                                                                                                  |
```
Comment (iamanmolm) [inline docs/cloud-cost-management/get-started/onboarding-guide/set-up-cost-visibility-for-aws.md:147]: Change the resources you had earlier seems correct only EC2, EBS, RDS, S3.

File: docs/cloud-cost-management/get-started/onboarding-guide/set-up-cost-visibility-for-azure.md [hunk 4]
```diff
@@ -148,7 +148,13 @@ CCM offers the following features:
 | **Cost Visibility** (Required)| This feature is available by default and requires access to the billing export. Provides the following functionalities:<ul><li>Insights into Azure costs by services, accounts, etc.</li><li>Root cost analysis using cost perspectives</li><li>Cost anomaly detection</li><li>Governance using budgets and forecasts</li><li>Alert users using Email and Slack notification</li></ul>|
 | **Azure Inventory Management** (Optional)| This feature provides visibility into your Azure VM inventory dashboard and metrics dashboard. The insights provided by inventory management can be used by finance teams to understand resource utilization across the board.|
 | **Azure optimization using AutoStopping rules** (Required for AutoStopping Rules)| This feature allows you to enable Intelligent Cloud AutoStopping for your Azure instances with a simple one-time setup. For more information, go to **Create AutoStopping Rules for Azure**.<ul><li>Orchestrate GCE VMs based on idleness</li><li>Set dependencies between VMs</li><li>Granular savings visibility</li><li>Simple one-time setup</li></ul>|
+<<<<<<< Updated upstream
```
Comment (iamanmolm) [inline docs/cloud-cost-management/get-started/onboarding-guide/set-up-cost-visibility-for-azure.md:151]: You also have this unresolved conflict, kindly fix this.

File: docs/cloud-cost-management/get-started/onboarding-guide/set-up-cost-visibility-for-azure.md [hunk 5]
```diff
@@ -148,7 +148,13 @@ CCM offers the following features:
...
+| **Cloud Governance** (Optional)              | This feature allows you to optimize your cloud spend and avoid unnecessary costs by rightsizing resources and decommissioning unused instances. For more information, see [Asset governance](../../5-use-ccm-cost-governance/asset-governance/1-asset-governance.md). <ul><li>Asset Management (EC2, EBS, RDS, S3)</li><li>Automated Actions</li></ul>
```
Comment (iamanmolm) [inline docs/cloud-cost-management/get-started/onboarding-guide/set-up-cost-visibility-for-azure.md:152]: For this change resources to vm, disk, sqlserver, loadbalancer

File: docs/cloud-cost-management/get-started/onboarding-guide/set-up-cost-visibility-for-azure.md [hunk 6]
```diff
@@ -148,7 +148,13 @@ CCM offers the following features:
...
+<<<<<<< Updated upstream
+| **Cloud Governance** (Optional)              | This feature allows you to optimize your cloud spend and avoid unnecessary costs by rightsizing resources and decommissioning unused instances. For more information, see [Asset governance](../../5-use-ccm-cost-governance/asset-governance/1-asset-governance.md). <ul><li>Asset Management (EC2, EBS, RDS, S3)</li><li>Automated Actions</li></ul>   
 
+=======
+| **Cloud Governance** (Optional)              | This feature allows you to optimize your cloud spend and avoid unnecessary costs by rightsizing resources and decommissioning unused instances. For more information, see [Asset governance](../../5-use-ccm-cost-governance/asset-governance/1-asset-governance.md). <ul><li>Asset Management (EC2, EBS, RDS, Azure VM, GCP instance, S3)</li><li>Automated Actions</li></ul>   
+ 
+>>>>>>> Stashed changes
```
Comment (iamanmolm) [inline docs/cloud-cost-management/get-started/onboarding-guide/set-up-cost-visibility-for-azure.md:157]: Also I can see your stashed changes here in the PR.

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/overview.md
Comment (iamanmolm) [inline docs/cloud-cost-management/4-use-ccm-cost-optimization/commitment-orchestrator/overview.md:1]: How come CO came in this PR?

File: docs/cloud-cost-management/get-started/onboarding-guide/set-up-cost-visibility-for-gcp.md [hunk 7]
```diff
@@ -113,6 +113,7 @@ CCM offers the following features:
...
 | **GCP Inventory Management** (Optional)| This feature provides visibility into your GCE VMs and unused disks and snapshots. The insights provided by inventory management can be used by Finance teams to understand resource utilization across the board.|
 | **GCP optimization using AutoStopping rules** (Required for AutoStopping Rules)| This feature allows you to enable Intelligent Cloud AutoStopping for your GCP cloud resources. For more information, see **Create AutoStopping Rules for GCP**.<ul><li>Orchestrate GCE VMs based on idleness</li><li>Set dependencies between VMs</li><li>Granular savings visibility</li><li>Simple one-time setup</li></ul>|
+| **Cloud Governance** (Optional)              | This feature allows you to optimize your cloud spend and avoid unnecessary costs by rightsizing resources and decommissioning unused instances. For more information, see [Asset governance](../../5-use-ccm-cost-governance/asset-governance/1-asset-governance.md). <ul><li> Asset Management (EC2, EBS, RDS, Azure VM, GCP instance, S3) </li><li>Automated Actions</li></ul>
```
Comment (iamanmolm) [inline docs/cloud-cost-management/get-started/onboarding-guide/set-up-cost-visibility-for-gcp.md:116]: For this change resources to instance, disk, sql-instance, image

**Review by iamanmolm — CHANGES_REQUESTED** [review]

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md (hunk 1 above)
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/1-asset-governance.md]: Added

File: docs/cloud-cost-management/get-started/onboarding-guide/set-up-cost-visibility-for-azure.md (hunk 2 above)
Comment (joeyouss) [reply docs/cloud-cost-management/get-started/onboarding-guide/set-up-cost-visibility-for-azure.md:152]: Added

**Review by iamanmolm — APPROVED** [review]

**Comment by Rohit-Kaliki** [comment]
https://65ba3651cb39f5caad497b78--harness-developer.netlify.app/docs/cloud-cost-management/use-ccm-cost-governance/asset-governance/asset-governance#cloud-asset-governance-for-aws We should also add Azure and GCP permissions in this page. Other aspects look good to me.

**Comment by Rohit-Kaliki** [comment]
@joeyouss Going forward can we keep PRs separate for each unique change as much as possible, I see we have multiple changes factored into this PR>

**Pull request merged**